### ISSUE TITLE
Base class balance rework

### ISSUE THREAD

**Issue by Tedster59** [Tedster59/lwotc#10]
expand XCOM decks, tweak some ability stats as needed, redo trees as needed

**Comment by nalddox** [comment]
I don't know how the community feels about this, but my personal opinion on XCOM decks is that there is already too much dilution resulting in some XCOM decks feeling terrible/useless/unimpactful vs some XCOM deck abilities being absolutely crazy (e.g., Shadowstrike + Saturation Fire Gunner). Why not have the XCOM decks become build enablers rather than lotteries, influencing the decision to choose some of the lesser-used builds by having the capability to increase effectiveness with more carefully curated XCOM deck rolls?

**Comment by Tedster59** [comment]
Yep, that is one thing I want to consider doing, adding more relevant perks that might push one way or another, part of the issue is many cool perks I want to add aren't part of LWoTC by default, so I would need to get permission to include them, or make my own.
